images/devcontainer/root/etc/nix/nix.conf

```diff
@@ -0,0 +1 @@
+sandbox = false
```

Comment:
How about using `sandbox = relaxed` instead? Removing sandboxing entirely tends to hide some packaging issues.

Reply:
Indeed it works with `sandbox=relaxed`.